main/2025/CG-02.md

```diff
@@ -130,12 +198,2205 @@ sessions](https://calendar.google.com/calendar/u/0?cid=Y18zYmNkNzdiOWZmZjM4YzA0Y
+
+The developer-guide page isn’t very pretty, but has a lot of info.
+
+I went through all the open PRs, and I have now closed all of them.
+
+The same for the issues. Lots closed, only 2 more open.
+
+The two issues that are left open:
+No more-wasm tutorials. Turns out, there’s a lot of content on the Web. learn-wasm.org and other sites. If we find someone who wants to help us write content here, we can probably work with them to make that happen.
+
+Mendy: I wrote a lot of the MDN docs on Wasm.
+
+AI (in chat): The table should not be restricted by arbitrary max-width... If someone has a wide enough monitor, the whole table can't be seen because of the max-width: 1080px … fortunately, ctrl+shift+i exist
+
+TL (in chat): Now that we're past that part of the presentation, it would be great to open an issue about this.
+
+MF (in chat): or a PR!
+
+TS: Some of the less visible pages are actually hosted in the design repo. Someone from the community made a page to surface a hand-selected collection of design repo docs for the website. Open discussion for the community: what should we do with these pages?
+
+TS: WebAssembly has many different audiences coming from different backgrounds. People hearing about the “Web” in the name may assume it’s like JS or make other assumptions.
+
+TS: webassembly.org is the landing pad for most of the “WebAssembly is” hyperlinks around the Web. A lot of people copy-and-paste the opening sentence on the main page. What information should we present on the main page? I left most of this content as-is, so let’s have an open discussion. What should we do here?
+
+N: I visited the site and clicked on the FAQ page and it talks about how threads and SIMD are coming in the future, so it’s very out of date.
+
+TS: The question is, do we even need this?
+
+RH: Thank you for doing all this work. Taking a slight step back, I’m curious if there’s interest in a smaller group of people meeting regularly to work on updated content here. THey could perhaps be empowered to make changes, and check in with the CG every quarter perhaps, so this could be relatively low-effort.
+
+TS: Yes, I think this makes a lot of sense. 
+
+DanG: I've been going through the design repo and updating design.md and friends, so most of those are up to date now.
+	
+DanG: It looks like the git submodule in the website repo pointing to the design repo is three years old, so I will submit a PR to update that. Then the website will have up-to-date contents.
+
+FM: It would be nice to have guidelines for people who wish to have their site linked to from the webassembly.org page. Then we could make it easier for people to contribute to the website.
+
+(discussion of approaches to guidelines)
+
+TL: Perhaps we could use our unconference time to discuss this further.
+
+### Thursday, February 13
+
+#### WebAssembly Memory & JSArrayBuffers (Deepti Gandluri)
+
+Slides: TODO
```

Comment:
@dtig, can you provide a link to your slides?

Reply:
Added slides directly to the notes.